packages/react-core/src/demos/Animations/examples/Animations.tsx

```diff
@@ -0,0 +1,1699 @@
+import imgAvatar from '@patternfly/react-core/src/components/assets/avatarImg.svg';
+import SearchIcon from '@patternfly/react-icons/dist/esm/icons/search-icon';
+import pfLogo from '@patternfly/react-core/src/demos/assets/pf-logo.PF-HorizontalLogo-Color.svg';
+import MultiContentCard from '@patternfly/react-component-groups/dist/dynamic/MultiContentCard';
+import { applicationsData } from './ResourceTableData';
+import SkeletonTable from '@patternfly/react-component-groups/dist/dynamic/SkeletonTable';
+
+export const Animations: FunctionComponent = () => {
+  const drawerRef = useRef<HTMLElement | null>(null);
+
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+  const [isKebabDropdownOpen, setIsKebabDropdownOpen] = useState(false);
+  const [isDrawerExpanded, setIsDrawerExpanded] = useState(false);
+  const [isAlertVisible, setIsAlertVisible] = useState(false);
+  const [isAlert2Visible, setIsAlert2Visible] = useState(false);
+  const [isAlert3Visible, setIsAlert3Visible] = useState(false);
+  const [showForm, setShowForm] = useState(false);
+  const [alert1Id, setAlert1Id] = useState('');
+  const [alert2Id, setAlert2Id] = useState('');
+  const [alert3Id, setAlert3Id] = useState('');
+
+  interface UnreadMap {
+    [notificationId: string]: boolean;
+  }
+
+  const [activeItem, setActiveItem] = useState<number | string>(0);
+  const [activeGroup, setActiveGroup] = useState<string | null>(null);
+  const [isUnreadMap, setIsUnreadMap] = useState<UnreadMap | null>({
+    'notification-1': true,
+    'notification-2': true,
+    'notification-3': false,
+    'notification-4': false
+  });
+
+  const [shouldShowNotifications, setShouldShowNotifications] = useState(true);
+
+  interface ActionsMenu {
+    [toggleId: string]: boolean;
+  }
+
+  const handleShowForm = () => {
+    setShowForm(!showForm);
+  };
+
+  const [isActionsMenuOpen, setIsActionsMenuOpen] = useState<ActionsMenu | {}>({});
+
+  const [selectedTab, setSelectedTab] = useState(0);
+
+  const [shouldNotify, setShouldNotify] = useState(false);
+  const prevUnreadCountRef = useRef(0);
+
+  const getNumberUnread = () => {
+    if (!isUnreadMap) {
+      return 0;
+    }
+    return Object.values(isUnreadMap).filter(Boolean).length;
+  };
+
+  const onNavSelect = (
+    _event: FormEvent<HTMLInputElement>,
+    selectedItem: {
+      groupId: number | string | null;
+      itemId: number | string;
+      to: string;
+    }
+  ) => {
+    setActiveItem(selectedItem.itemId);
+    setActiveGroup(selectedItem.groupId as string | null);
+  };
+
+  const onDropdownToggle = () => setIsDropdownOpen((prevState) => !prevState);
+  const onDropdownSelect = () => setIsDropdownOpen(false);
+  const onKebabDropdownToggle = () => setIsKebabDropdownOpen((prevState) => !prevState);
+  const onKebabDropdownSelect = () => setIsKebabDropdownOpen(false);
+  const onCloseNotificationDrawer = (_event: any) => setIsDrawerExpanded((prevState) => !prevState);
+
+  useEffect(() => {
```

Comment:
Also just on a more general note my gut says that the way that all of these alerts are being made and managed could be DRY'd up substantially, but since this _is_ just a demo I'm not _too_ worried about that.